incubator/nifi/values.yaml

```diff
@@ -0,0 +1,214 @@
+# Default values for NiFi.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+## Use ClusterIP if your setup includes ingress controller
+serviceType: LoadBalancer
+
+## Configure Ingress resource that allow you to access the Drupal installation
+## Set up the URL
+## ref: http://kubernetes.io/docs/user-guide/ingress/
+ingress:
+  enabled: true
+  domain: nifi.govcloud.ca
```

Comment:
I imagine typical users of this chart cannot use the domain `nifi.govcloud.ca`. Is there another way to handle this than to have your domain? Possible `example.com` which is a reserved name.